api/models/joi-schemas.ts

```diff
@@ -10,6 +10,11 @@ export const createLinkSchema = Joi.object({
     )
     .required()
     .trim(),
+  customCode: Joi.string()
+    .regex(
+      /^(?=.*[A-Za-z])[^-_][-\w]{3,75}$/
```

Comment:
Why two different types of validation? 4 and 120 and 3 and 75?

Reply:
The custom code must contain **at least** one letter and three other characters, in total 4 and it must be less than 75.